node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/NodeSpec.java

```diff
@@ -526,6 +538,11 @@ public Builder type(NodeType nodeType) {
             return this;
         }
 
+        public Builder cloudAccount(CloudAccount cloudAccount) {
+            this.cloudAccount = cloudAccount;
```

Comment:
requireNonNull here?

Reply:
We don't have any validation in any other builder methods in this class and instead rely on the validation in the main constructor